superset/assets/javascripts/modules/AnnotationTypes.js

```diff
@@ -0,0 +1,29 @@
+import { VIZ_TYPES } from '../../visualizations/main';
+
+const ANNOTATION_TYPES = {
+  FORMULA: 'FORMULA',
+  EVENT: 'EVENT',
+  INTERVAL: 'INTERVAL',
+  TIME_SERIES: 'TIME_SERIES',
+  NATIVE: 'NATIVE',
+//  POINT_ANNOTATION: 'POINT_ANNOTATION',
+};
+
+export function supportedSliceTypes(annotationType) {
+  if (annotationType === ANNOTATION_TYPES.EVENT ||
+      annotationType === ANNOTATION_TYPES.INTERVAL
+  ) return [VIZ_TYPES.table];
+  if (annotationType === ANNOTATION_TYPES.TIME_SERIES) return [VIZ_TYPES.line];
+  return [];
+}
+
+export function isQueryAnnotationType(annotationType) {
```

Comment:
this is a confusing name, what's a "query annotation"? do you mean `annotationRequiresQuery`? that's more clear to me.